_savedatasets.py

```diff
@@ -74,6 +74,16 @@
         metric='minute',
         considerFrequency=True
     ),
+    lastfm=dict(
+        gender='_setup.GENDER',
+        pastVerb='listened to',
+        pastVerb_2='listened to music for',
```

Comment:
How is pastVerb_2 is used? Can you also send a pull request to [this repo](https://github.com/tanykim/quantify-your-year) that uses this part?